package.json

```diff
@@ -5,7 +5,7 @@
   "version": "4.0.4",
   "repository": {
     "type": "git",
-    "url": "git://github.com/form-data/form-data.git"
+    "url": "https://github.com/form-data/form-data.git"
```

Comment:
git+ssh instead please

Reply:
Now changed to `ssh://` per https://github.com/inspect-js/has-symbols/pull/22#discussion_r2274665816